src/Logic/DarkTheme.cs

```diff
@@ -56,19 +57,30 @@ private static List<T> GetSubControls<T>(Control c)
             return menus.Cast<T>().ToList();
         }
 
-        public static void SetDarkTheme(Control form, int iterations = 5)
+        private static bool _isConfigUpdated;
+
+        public static void SetDarkTheme(Control ctrl, int iterations = 5)
```

Comment:
Rename **form** => **ctrl** see: https://github.com/SubtitleEdit/subtitleedit/pull/2816/files#diff-4e0c1aa2089eeedfb01eba69172efb07L69

it doesn't really make sense saying ` if (form is Form)` so **form** is now renamed to **ctrl** `if (ctrl is Form)`